homeassistant/components/shelly/climate.py

```diff
@@ -16,7 +16,7 @@
     HVACMode,
 )
 from homeassistant.config_entries import ConfigEntry
-from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS
+from homeassistant.const import ATTR_TEMPERATURE, TEMP_CELSIUS, UnitOfTemperature
```

Comment:
I am ok with merging this into RC if we need CI to pass, but it would better be to align it with dev
```suggestion
from homeassistant.const import ATTR_TEMPERATURE, UnitOfTemperature
```
And fix line 113 to use `UnitOfTemperature.CELSIUS`